ledger-core/virtual/integration/utils/server.go

```diff
@@ -152,6 +152,7 @@ func newServerExt(ctx context.Context, t Tester, errorFilterFn logcommon.ErrorFi
 	s.pulseManager = PulseManager
 	s.pulseStorage = Pulses
 	s.pulseGenerator = testutils.NewPulseGenerator(10)
+	s.incrementPulse(ctx) // for sake of simplicity make sure that there is no "hanging" first pulse
```

Comment:
Can we move that part to init?

Reply:
this one could be removed, because in Init we already do IncerementPulse

Reply:
we need 2 increments (first will have no valid prev)

Reply:
moved to init